zerver/openapi/zulip.yaml

```diff
@@ -1225,657 +1263,3511 @@ paths:
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to specific users on special cases meant
+                                to be displayed to the user in a type of alert box.
+                              properties:
+                                id:
+                                  type: integer
+                                type:
+                                  type: string
+                                  enum:
+                                    - hotspots
+                                hotspots:
+                                  type: array
+                                  description: |
+                                    An array of dictionaries where each
+                                    dictionary contains details about a single hotspot.
+                                  items:
+                                    $ref: "#/components/schemas/Hotspot"
+                              example:
+                                {
+                                  "type": "hotspots",
+                                  "hotspots":
+                                    [
+                                      {
+                                        "name": "intro_streams",
+                                        "title": "Catch up on a stream",
+                                        "description": "Messages sent to a stream are seen by everyone subscribed to that stream. Try clicking on one of the stream links below.",
+                                        "delay": 0.5,
+                                      },
+                                    ],
+                                  "id": 0,
+                                }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent when a message has been edited.
+                                Sent to all users who had received the original
+                                message.
+                              properties:
+                                id:
+                                  type: integer
+                                type:
+                                  type: string
+                                  enum:
+                                    - update_message
+                                user_id:
+                                  type: integer
+                                  description: |
+                                    The id of the user who sent the message.
+                                sender:
+                                  type: string
+                                  description: |
+                                    The email of the user who sent the message.
+                                    Only present in some cases.
```

Comment:
Looking at the implementation, I don't think this should appear and is likely a bug.